### DISCUSSION TITLE
Honeygain : using JMPT token tutorial

### DISCUSSION THREAD

**Discussion by OlivierGaland** [OlivierGaland/CashFactory#14] [Show and tell]
! Note : this is actually a draft, I will update it to make it more clear.

Honeygain propose 2 ways to earn money :
- Standard one with $20 payout (paypal,btc) , this takes time to reach the $20 payout (around 2 and a half months for me)
- Jumptask wallet (With apparently a +50% credit bonus and no minimal payout)

Note that those 2 wallet are not connected, you cannot transfer from one to another, as a consequence, when they deployed the Jumptask wallet, I had $13 on the standard wallet and had to continue until I got $20 to get my payout.

I tried both (the standard $20 and around $4.5 with JMPT token) and they works. This article will try to explain how to deal with JMPT token to convert them to money. Up to you to choose which way is better for you. I will use JMPT from now.

But this need intermediate skills in cryptocurrencies :

The issue : 
JMPT is a secondary cryptocurrency, only traded using pancakeswap. To turn it into money, you must trade it for a well known cryptocurrency, then send it to an exchange where you will be able to do what you want with (store, sell for money and get it back on your bank account)

Prerequisites :
- A account on an exchange listing BNB cryptocurrency (the exchange binance is the best candidate as it is a crypto they created and is used for fees on their platform)
- An installed Metamask wallet , you should configure BSC network on it (TODO for configuration)

Procedure to convert JMPT from honeygain wallet to a more known cryptocurrency (I choosed BNB) :

A1) Buy on your exchange some BNB, it will be used as fee for pancakeswap
A2) Send those BNB on your metamask Wallet
This step could be done one time only, you just need a few BNB on Metamask to cover swap fees, and as you convert JMPT to BNB, you will increase your BNB count (just don't send all back to your exchange)

B1) From Honeygain dashboard click to go on Jumptask dashboard
B2) Once on Jumptask dashboard, link your Metamask wallet and transfer your JMPT token on it (note there is a fee at this step)
B3) Go to https://pancakeswap.finance and connect your metamask wallet (if not already connected)
B4) Choose trade -> swap and do a swap from JMPT to BNB (there will be a fee at this step)
B5) Once the swap is done you can send back your BNB (not all, keep a few for the next payout) to your exchange
B6) do what you want : keep your BNB, buy bitcoin or buy $ and ask the exchange to send money on your credit card (there will be fee)

I did the procedure today with around 5000 credits on jumptask wallet and here is the transactions I had : 

Step A2 (Initial deposit BNB to cover swap fees) : 
Coin BNB
Withdraw amount 0.0095 ($3.91)
Network fee 0.0005 ($0.21) => fixed fee on binance

Step B4 (Turning JMPT into BNB) : 
From XXX To PancakeSwap V2: JMPT-BUSD For 2.2226178396758046 ($4.56) JumpToken (JMPT)
From PancakeSwap V2: JMPT-BUSD To YYY For 4.563452682155576326 ($4.57) Binance-Peg ... (BUSD)
From YYY To PancakeSwap: Router v2 For 0.011085823180939803 ($4.56) Wrapped BNB (WBNB)
Transaction Fee: 0.00087564 BNB ($0.36) => this is the BNB provided with step A)

Step B5 (Send BNB to my exchange):
Value: 0.01 BNB ($4.11)
Transaction Fee: 0.000105 BNB ($0.04)

Basically my $4.60 of JMPT were turned into $4.1 of BNB (fees were around 10%, they will decrease if you trade higher amount). I did it as a test and probably will do the procedure when I have around $10 of JMPT
